cleanrl/td3_continuous_action_jax.py

```diff
@@ -263,28 +270,28 @@ def actor_loss(params):
                 [
                     (
                         jax.device_get(actions)[0]
-                        + np.random.normal(0, max_action * args.exploration_noise, size=envs.single_action_space.shape[0])
+                        + np.random.normal(0, max_action * args.exploration_noise, size=envs.single_action_space.shape)
```

Comment:
Is this a problem?

Reply:
Yeah, I should have made a note, this would allow weirder action spaces